### ISSUE TITLE
Dask Dashboard not working on AML Compute Instance (CI)

### ISSUE THREAD

**Issue by lostmygithubaccount** [dask/dask-cloudprovider#126]
**What happened**:
Dask Dashboard shows no data below the top toolbar when using AzureMLCluster on a Compute Instance.

**What you expected to happen**:
Can view the Dask Dashboard. 

**Minimal Complete Verifiable Example**:
Start AzureMLCluster on a CI and click the Dask dashboard link.

```python
from azureml.core import Workspace
from dask_cloudprovider import AzureMLCluster

ws = Workspace.from_config()
cluster = AzureMLCluster(ws)
cluster
```

**Anything else we need to know?**:

**Environment**:
CI

- Dask version:
- Python version:
- Operating System:
- Install method (conda, pip, source):

**Comment by arnabbiswas1** [comment]
Just to add to this bug. The dashboard looks the following:

![image](https://user-images.githubusercontent.com/4428191/96122387-09e71f80-0f0f-11eb-8206-fae0e46813ed.png)

Only the Info tab shows the following:

![image](https://user-images.githubusercontent.com/4428191/96122478-300cbf80-0f0f-11eb-85dd-3343484c3b1e.png)

Clicking on the Dashboard gives the following error:

> Try navigating to 10.233.0.7:42295/status for your worker dashboard
> 
> Dask tried to proxy you to that page through your Scheduler's dashboard connection, but you don't have jupyter-server-proxy installed. You may want to install it with either conda or pip, and then restart your scheduler.
> 
>  conda install jupyter-server-proxy -c conda-forge 
>  python -m pip install jupyter-server-proxy
> The link above should work though if your workers are on a sufficiently open network. This is common on single machines, but less common in production clusters. Your IT administrators will know more

Clicking on the hyperlink under Worker column & Logs gave 404.

**Comment by mrocklin** [comment]
Workers are rarely exposed to the public internet when deploying on cloud
providers.  (exposing IP addresses is expensive).

You might consider using the instructions there to install the jupyter
server proxy

**Comment by arnabbiswas1** [comment]
@mrocklin I tried this. But it didn't help. 

@lostmygithubaccount Are you able to create a client here on AML Compute instance using the cluster created:

```
from dask.distributed import Client

c = Client(amlcluster)
c
```

**Comment by arnabbiswas1** [comment]
> @lostmygithubaccount Are you able to create a client here on AML Compute instance using the cluster created:
> 
> ```
> from dask.distributed import Client
> 
> c = Client(amlcluster)
> c
> ```

As found in #145, creation of client was failing because of a version issue. Nothing to do with connectivity.

**Comment by jacobtomlinson** [comment]
Closed by #166
